### PR TITLE
Implementation of new snackbar component

### PR DESCRIPTION
**Pull request by matgitirana**
Implementation of new snackbar component, which can show toast style notifications.

### CODE REVIEW

File: src/components/snackbars/SnackbarNotification.tsx [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+export function SnackbarContentWrapper(props: SnackbarContentProps) {
+    const classes = useStyles();
+    const { message, onClose, variant } = props;
+    const Icon = variantIcon[variant];
+
+    return (
+        <SnackbarContent
+            className={classes[variant]}
+            message={
+                <span className={classes.message}>
```
Comment (DaniloArruda) [inline src/components/snackbars/SnackbarNotification.tsx]: modify to const

Comment (matgitirana) [reply src/components/snackbars/SnackbarNotification.tsx]: resolved

File: src/components/snackbars/SnackbarNotification.tsx [hunk 2]
```diff
@@ -0,0 +1,93 @@
+import React, { SyntheticEvent } from 'react';
+import { Snackbar, SnackbarContent, IconButton } from '@material-ui/core';
+import ErrorIcon from '@material-ui/icons/Error';
+import { makeStyles } from '@material-ui/styles';
+import clsx from 'clsx';
+import CloseIcon from '@material-ui/icons/Close';
+
+export interface SnackbarNotificationProps {
+    variant: 'error';
```
Comment (DaniloArruda) [inline src/components/snackbars/SnackbarNotification.tsx]: Set variant type as string to have the possibility to define other variants such as 'success' and 'warn'

File: src/components/snackbars/SnackbarNotification.tsx [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+    message: string;
+    open: boolean;
+    handleClose: () => void;
+}
+
+export default function SnackbarNotification(props: SnackbarNotificationProps) {
+    const { variant, message, open, handleClose } = props;
+
+    function onClose(event?: SyntheticEvent, reason?: string) {
```
Comment (DaniloArruda) [inline src/components/snackbars/SnackbarNotification.tsx]: Use arrow function syntax: const onClose = (parameters) => {...}

File: src/components/snackbars/SnackbarNotification.tsx [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+
+interface SnackbarContentProps {
+    className?: string;
+    message?: string;
+    onClose?: () => void;
+    variant: keyof typeof variantIcon;
+}
+
+const variantIcon = {
+    error: ErrorIcon,
```
Comment (DaniloArruda) [inline src/components/snackbars/SnackbarNotification.tsx]: Add the other variants: success, warn, and info

File: src/components/snackbars/SnackbarNotification.tsx [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+                </IconButton>,
+            ]}
+        />
+    );
+}
+
+const useStyles = makeStyles({
+    error: {
+        backgroundColor: '#D32F2F',
+    },
```
Comment (DaniloArruda) [inline src/components/snackbars/SnackbarNotification.tsx]: Add the other variants: success, warn, and info

Comment (matgitirana) [reply src/components/snackbars/SnackbarNotification.tsx]: Done

File: src/components/snackbars/SnackbarNotification.tsx (hunk 4 above)
Comment (matgitirana) [reply src/components/snackbars/SnackbarNotification.tsx]: Done

Comment (matgitirana) [reply src/components/snackbars/SnackbarNotification.tsx]: Done

File: src/components/snackbars/SnackbarNotification.tsx (hunk 2 above)
Comment (matgitirana) [reply src/components/snackbars/SnackbarNotification.tsx]: Done

File: src/components/snackbars/SnackbarNotification.tsx (hunk 3 above)
Comment (matgitirana) [reply src/components/snackbars/SnackbarNotification.tsx]: Done

**Pull request merged**